ecs-image-build/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG IMAGE_VERSION="latest"
+FROM 416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-node-build-14:${IMAGE_VERSION} 
+
+ARG APPLICATION_NAME=applications.developer.web.ch.gov.uk
+ARG PORT="3000"
+ARG NODE_PORT="3000"
```

Comment:
These build args aren't being used in the Dockerfile at all. Should they be getting assigned to ENVs?